modules/consensus/accept_test.go

```diff
@@ -1349,7 +1351,7 @@ func (cst *consensusSetTester) testPaymentChannelBlocks() error {
 		// txn needs to be fully custom. To get a custom txn, manually create an
 		// address and then use the wallet to fund that address.
 		channelSize := types.NewCurrency64(10e3)
-		channelFundingSK, channelFundingPK, err := crypto.GenerateSignatureKeys()
```

Comment:
I'm not sure if we're talking about the same thing. It sounds like you're advocating this:

```
type SKGClient struct {
  // some members
}

func (c SKGClient) Foo() {
  sk, pk, err := crypto.GenerateSignatureKeys()
  ...
}
```

How do we test `SKGClient.Foo()` in that case? It's tightly bound to `crypto.GenerateSignatureKeys` so we can't mock it out.

I'm saying I'd ultimately like to get to this:

```
type SKGClient struct {
  // some members
  signatureKeyGenerator KeyPairGenerator
}

func (c SKGClient) Foo() {
  sk, pk, err := signatureKeyGenerator.Generate()
  ...
}
```

That way we can easily test `SKGClient` by initializing it with a mock object for `signatureKeyGenerator `.

I prefer what's currently in the PR because it's closer to this ultimate goal.